src/implementations/twig/pages/page-home/page-home.html.twig

```diff
@@ -0,0 +1,75 @@
+<!-- Site header -->
+{%- include '@ecl/site-header/site-header.html.twig' with site_header -%}
+
+<main>
+  <div class="ecl-container">
+    <!-- Banner -->
+    {%- include '@ecl/banner/banner.html.twig' with banner -%}
+
+    <h2 class="ecl-u-type-heading-2">Maecenas sagittis est</h2>
```

Comment:
i noticed that we are not doing what i thought with the margins we added in the default css.
We set the margin for any h2 that contains an ecl utility class: .ecl h2:is([class*=ecl-u-] and i don't understand why.
It shouldn't be the case, this way we are basically adding this margin to any utility class used on an h2 inside an .ecl container and instead we should only add it in case no utility is set, just a generic h2, otherwise the reasoning is that if a utility is used, this utility might be already setting a margin or if it's not, another utility class can be added to set a margin on it.

Reply:
So basically we should have something like
```
.ecl h2:not([class*='wt-']) {
  @extend %ecl-heading-2;
}
```
I also noticed that the default style for the separator is not correct either; it would stop to work as soon as we add any utility to it.
I'll create a separated ticket to handle all of that

Reply:
mmh..it might be that we have some misunderstanding about how these selectors should be defined based on what they are supposed to do: my approach is that if a user is using an ecl-utility that is not "generic" html, if there is an ECL selector we might assume that the styles are already handled by those, also because we cannot know which utility is being used and having two diferent styles set using !important would really create an unpredictable situation.
Maybe i haven't shared these thoughts about this css in a clear way before, but that is the idea, generic means something that doesn't use ECL already, imo.
Open to different considerations, also because we are distributing this css since v4 so at this point better not altering the existing behaviors, but for the future maybe this can be taken into account.